enhancements/rhcos/rhcos-cloud-marketplace-images.md

```diff
@@ -0,0 +1,330 @@
+x64             rh-ocp-worker  redhat-rhel     rh-ocp-worker       redhat-rhel:rh-ocp-worker:rh-ocp-worker:4.13.2023101100            4.13.2023101100
+x64             rh-ocp-worker  redhat-limited  rh-ocp-worker-gen1  redhat-limited:rh-ocp-worker:rh-ocp-worker-gen1:4.8.2021122100     4.8.2021122100
+x64             rh-ocp-worker  redhat-limited  rh-ocp-worker-gen1  redhat-limited:rh-ocp-worker:rh-ocp-worker-gen1:413.92.2023101700  413.92.2023101700
+x64             rh-ocp-worker  redhat-rhel     rh-ocp-worker-gen1  redhat-rhel:rh-ocp-worker:rh-ocp-worker-gen1:4.13.2023101100       4.13.2023101100
+```
+
+Images from the `redhat-limited` publisher are for EMEA while `RedHat` is for North America. Free marketplace images could be published under
+a new offer, such as: `RHCOS`, `OpenShift` or, keeping in line with the current convention, `rh-ocp`, such that the listing would look like:
+
+```shell
+$  az vm image list --all --offer rhcos --publisher redhat -o table
+Architecture    Offer          Publisher       Sku                 Urn                                          Version
+--------------  -------------  --------------  ------------------  -------------------------------------------  -----------------
+x64             rhcos          RedHat          rhcos               RedHat:rhcos:rhcos:417.94.202407010929       417.94.202407010929
+x64             rhcos          RedHat          rhcos-gen1          RedHat:rhcos:rhcos-gen1:417.94.202407010929  417.94.202407010929
+```
+
+##### Azure Stream Representation
+
+An Azure Marketplace extension, `azure-marketplace`, would be added, with unpaid and paid (both North America & EMEA)
+marketplace images listed as child objects: 
+
+```yaml
+"rhel-coreos-extensions": {
+  "azure-disk": {
+    "release": "417.94.202407010929-0",
+    "url": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-417.94.202407010929-0-azure.x86_64.vhd"
+  },
+  "azure-marketplace": {
+    "no-purchase-plan": {
+      "publisher": "RedHat",
+      "offer": "rhcos",
+      "sku": "rhcos",
+      "version": "417.94.202407010929"
+    },
+    "purchase-plan-north-america": {
+      "publisher": "RedHat",
+      "offer": "rh-ocp-worker",
+      "sku": "rh-ocp-worker",
+      "version": "417.94.202407010929"
+    },
+    "purchase-plan-emea": {
+      "publisher": "redhat-limited",
+      "offer": "rh-ocp-worker",
+      "sku": "rh-ocp-worker",
+      "version": "417.94.202407010929"
+    }
+  }
+```
+
+[Hyperv Generation 1 images](https://learn.microsoft.com/en-us/azure/virtual-machines/generation-2#features-and-capabilities)
+would be published, identified by a `gen1` suffix, but would not be included in the RHCOS stream. Any time that the installer
+or users need to select a gen1 image, they would simply add the `gen1` suffix to the SKU identified in the RHCOS stream.  
+
+OPEN QUESTION: How to handle EMEA vs North America 
```

Comment:
What's the purpose of EMEA vs. North America images?  The ARO Marketplace images don't make that distinction as far as i can tell.